website/docs/r/transport_node.html.markdown

```diff
@@ -0,0 +1,276 @@
+---
+subcategory: "Fabric"
+layout: "nsxt"
+page_title: "NSXT: nsxt_transport_node"
+description: A resource to configure a Transport Node.
+---
+
+# nsxt_transport_node
+
+This resource provides a method for the management of a Transport Node.
+This resource is supported with NSX 4.1.0 onwards.
+
+## Example Usage
+
+```hcl
+resource "nsxt_transport_node" "test" {
+  description  = "Terraform-deployed edge node"
+  display_name = "tf_edge_node"
+  standard_host_switch {
+    host_switch_mode = "STANDARD"
+    host_switch_type = "NVDS"
+    ip_assignment {
+      static_ip_pool_id = data.nsxt_ip_pool.ipp1.id
```

Comment:
we should avoid suggesting deprecated objects in docs, but we can refine this later

Reply:
Can we use the policy IP pool in the MP transport node? How do we gather its id?

Reply:
Realization data source should provide the realized ID

Reply:
OK. As I'm not sure how to use realization info data source for IP pool, I'll use DHCP for the example :)